routes/web.php

```diff
@@ -53,8 +53,8 @@
 
 $router->group(['prefix' => 'reservations'], function () use ($router) {
     $router->get('search', ['uses' => 'ReservationController@search', 'middleware' => 'auth:reservation']);
-    $router->get('{id}', ['uses' => 'ReservationController@show', 'middleware' => 'auth:reservation']);
-    $router->get('{id}/check', ['uses' => 'ReservationController@check']);
+    $router->get('{id}', ['uses' => 'ReservationController@show', 'middleware'=>'throttle:5, 1']);
```

Comment:
スタイルの話なので "suggestion" に留めておきます
```suggestion
    $router->get('{id}', ['uses' => 'ReservationController@show', 'middleware' => 'throttle:5, 1']);
```